doc/MONSTERS.md

```diff
@@ -80,6 +80,8 @@ Monsters may also have any of these optional properties:
 | `revert_to_itype`        | (string) Item monster can be converted to when friendly (ex. to deconstruct turrets)
 | `starting_ammo`          | (object) Ammo that newly spawned monsters start with
 | `upgrades`               | (boolean or object) False if monster does not upgrade, or an object do define an upgrade
+| `zombify_into`           | (boolean or string) False if monster does not get zombified, or a string to define which monster it turns into when resurrected, by default it's false
```

Comment:
My bad for not documenting it,, but this is false zombify_into is not a boolean. It's either a mtyper_id or it's empty and by default it's empty.